docs/hub/gguf-llamacpp.md

```diff
@@ -1,16 +1,46 @@
 # GGUF usage with llama.cpp
 
-Llama.cpp directly allows you to download and run inference on a GGUF simply by providing a path to the Hugging Face repo path and the file name. llama.cpp would download the model checkpoint in the directory you invoke it from:
+Llama.cpp allows you to download and run inference on a GGUF simply by providing a path to the Hugging Face repo path and the file name. llama.cpp would download the model checkpoint in the directory you invoke it from:
 
 ```bash
-./main \
+./llama-cli \
   --hf-repo lmstudio-community/Meta-Llama-3-8B-Instruct-GGUF \
   -m Meta-Llama-3-8B-Instruct-Q8_0.gguf \
   -p "I believe the meaning of life is " -n 128
 ```
 
+Additionally, you can invoke an OpenAI spec chat completions endpoint directly using the llama.cpp server
+
+```bash
+./llama-server \
+  --hf-repo lmstudio-community/Meta-Llama-3-8B-Instruct-GGUF \
+  -m Meta-Llama-3-8B-Instruct-Q8_0.gguf \
+  -p "I believe the meaning of life is " -n 128
```

Comment:
no need for `-p` and `-n` for llama-server, since it can be controlled in the HTTP request.

The -p prompt is the system prompt, and -n number of tokens is mapped to `"max_tokens"` (see [OpenAI docs](https://platform.openai.com/docs/api-reference/chat/create))

Reply:
Good call! I was lazy and just copied the binaries lol.